packages/react-components/src/components/tests/TypingIndicator.test.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { TypingIndicator } from '../TypingIndicator';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { mountWithLocalization } from './enzymeUtils';
+import { act } from 'react-dom/test-utils';
+import englishStrings from '../../localization/translated/en-US.json';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+test('TypingIndicator test in english', async () => {
+  const component = mountWithLocalization(
+    <TypingIndicator typingUsers={[{ userId: 'user2', displayName: 'Claire' }]} />,
+    englishStrings
+  );
+  await act(async () => component);
+  component.update();
+  expect(component.html()).toBe(

```

Comment:
can we use toMatchSnapshot here? (or toMatchInlineSnapshot)? (might need an extra plugin to work with enzyme)

Reply:
https://www.npmjs.com/package/enzyme-to-json

Reply:
Sure I could do that. I just have one question. I have a ParticipantItem test example in this PR that uses jest snapshot just to show another option. Changes not related to localization to a component would cause the need for updating snapshots. Are we ok with that?

Reply:
Yep, ideally we should have snapshot tests for every component testing different props. I think a good goal would be unit test snapshots in the react-components folder, then integration tests in storybook instead of having snapshots in storybook